pkgs/development/libraries/sword/default.nix

```diff
@@ -34,9 +27,18 @@ stdenv.mkDerivation rec {
   meta = with stdenv.lib; {
     description = "A software framework that allows research manipulation of Biblical texts";
     homepage = "http://www.crosswire.org/sword/";
-    platforms = platforms.linux;
+    longDescription = ''
+      The SWORD Project is the CrossWire Bible Society's free Bible software
+      project. Its purpose is to create cross-platform open-source tools --
+      covered by the GNU General Public License -- that allow programmers and
+      Bible societies to write new Bible software more quickly and easily. We
+      also create Bible study software for all readers, students, scholars, and
+      translators of the Bible, and have a growing collection of many hundred
+      texts in around 100 languages.
+    '';
     license = licenses.gpl2;
-    maintainers = [ maintainers.piotr maintainers.AndersonTorres ];
+    maintainers = with maintainers; [ piotr AndersonTorres ];
+    platforms = platforms.linux;
```

Comment:
is that a fix ? or a workaround ? ihttp://www.crosswire.org/faq/#Questions_about_the_platforms_our_software_runs_upon seems to suggest a mac version is ok

Reply:
I didn't know, and I have no Mac to test it.

Maybe `unix` is OK here?

Reply:
I was just concerned mac users would be suprised not being able to run it anymore. Did it fail on the darwin CI ?